### ISSUE TITLE
SWIG failing

### ISSUE THREAD

**Issue by mpowelson** [ros-industrial-consortium/tesseract_ext#4]
I'm not sure why this is failing, but here is the error. This is on current master on 18.04 building with colcon
```
--- stderr: swig                               
Swig version found: 3.0.12
+ test -d Tools/config
+ aclocal -I Tools/config
+ autoheader
+ automake --add-missing --copy --force-missing
+ autoconf
+ cd CCache
+ autoreconf
make[3]: warning: -jN forced in submake: disabling jobserver mode.
../Tools/config/ylwrap: 176: ../Tools/config/ylwrap: yacc: not found
make[4]: *** [CParse/parser.c] Error 127
make[3]: *** [source] Error 2
make[2]: *** [swig-prefix/src/swig-stamp/swig-build] Error 2
make[1]: *** [CMakeFiles/swig.dir/all] Error 2
make: *** [all] Error 2
---
Failed   <<< swig	[ Exited with code 2 ]
```

**Comment by gavanderhoorn** [comment]
Seem to be missing `yacc`. On Ubuntu there are quite a few pkgs that could provide that tool. Not sure which one SWIG prefers though.

**Comment by Levi-Armstrong** [comment]
Addressed by #8
